libs/product/src/models/product.ts

```diff
@@ -25,19 +25,35 @@ export interface DaffProduct extends DaffLocatable {
 	/**
 	 * The type of product.
 	 */
-	type?: DaffProductTypeEnum;
-	/**
-	 * The price of the product.
-	 */
+  type?: DaffProductTypeEnum;
+  /**
+   * The product name.
+   */
+  name?: string;
+  /**
+   * The main image for the product.
+   */
+  thumbnail: DaffProductImage;
```

Comment:
I think the thumbnail should be described as "A smaller image to concisely describe the product". Or something like that.

Reply:
changed